### PR TITLE
Add Italian localization

### PR DESCRIPTION
**Pull request by BuriedInTheGround**
I've added the Italian localization.

Question: should we avoid accented letters and use a single quote instead?

### CODE REVIEW

**Comment by str4d** [comment]
Thanks! cc @FiloSottile for review.

> Question: should we avoid accented letters and use a single quote instead?

Nah; Rust supports UTF-8 natively, and I think it's fine to assume that if a terminal has Italian set as the language, it also has the character set to render it properly.

**Review by str4d — APPROVED** [review]
I didn't check the translations themselves, but I confirmed that they have the correct tags and placeholders.

**Comment by str4d** [comment]
Ooh, in testing this I found some more untranslated strings. I'll push a commit to this branch that extracts the `en-US` strings.

**Comment by str4d** [comment]
@BuriedInTheGround I've pushed some commits that extract more strings for translation that I'd missed.

File: age/i18n/it-IT/age.ftl [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+err-detected-binary = rilevati dati non stampabili; rifiuto l'invio dell'output al terminale.
+rec-detected-binary = Puoi forzarlo con '{-output-stdout}'.
+
+err-deny-binary-output = rifiuto l'invio di output binario al terminale.
+rec-deny-binary-output = Intendevi usare {-flag-armor}? {rec-detected-binary}
+
+## Errors
+
+err-decryption-failed = Decrittazione fallita
```
Comment (FiloSottile) [inline age/i18n/it-IT/age.ftl:37]: I think in general "decifrazione/cifrato/cifratura" is more common than "decrittazione/crittato/crittatura".

File: age/i18n/it-IT/age.ftl [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+err-excessive-work = Parametro di lavoro per la passphrase troppo elevato.
+rec-excessive-work = La decrittazione impiegherà circa {$duration} secondi.
+
+err-header-invalid = L'header è invalido
+
+err-header-mac-invalid = L'header MAC è invalido
```
Comment (FiloSottile) [inline age/i18n/it-IT/age.ftl:44]:
```suggestion
err-header-mac-invalid = Il MAC dell'header è invalido
```

File: age/i18n/it-IT/age.ftl [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+rec-unknown-format = Hai provato ad aggiornare all'ultima versione?
+
+## SSH identities
+
+ssh-passphrase-prompt = Inserisci la passphrase per la chiave OpenSSH '{$filename}'
+
+ssh-unsupported-identity = Identità SSH non supportata: {$name}
+
+ssh-insecure-key-format =
+    Formato della Chiave Crittografica Insicuro
```
Comment (FiloSottile) [inline age/i18n/it-IT/age.ftl:60]:
```suggestion
    Formato della Chiave Crittografica Non Sicuro
```

**Review by FiloSottile — APPROVED** [review]
LGTM with nits

File: age/i18n/it-IT/age.ftl (hunk 1 above)
Comment (BuriedInTheGround) [reply age/i18n/it-IT/age.ftl:37]: Ok! Initially I wasn't totally convinced, but I'll stick with your suggestion. I'll change these here and there.

File: age/i18n/it-IT/age.ftl (hunk 2 above)
Comment (BuriedInTheGround) [reply age/i18n/it-IT/age.ftl:44]: My bad, I'll fix it.

File: age/i18n/it-IT/age.ftl [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+## SSH identities
+
+ssh-passphrase-prompt = Inserisci la passphrase per la chiave OpenSSH '{$filename}'
+
+ssh-unsupported-identity = Identità SSH non supportata: {$name}
+
+ssh-insecure-key-format =
+    Formato della Chiave Crittografica Non Sicuro
+    -------------------------------------------
```
Comment (str4d) [inline age/i18n/it-IT/age.ftl:61]:
```suggestion
    Formato della Chiave Crittografica Non Sicuro
    ---------------------------------------------
```

**Comment by str4d** [comment]
Thanks!

**Pull request merged**